docs/developer-guide.md

```diff
@@ -24,16 +17,37 @@ All projects in Go must exist in the `$GOPATH`
 # Replace with https://github.com/grtl/mysql-operator for https
 ```
 
+## Install _godep_
+Install the lastest version via go get:
+```bash
+> go get github.com/tools/godep
+```
+
+Enable vendor:
+```bash
+> export GO15VENDOREXPERIMENT=1
+```
+
 ## Install dependencies
-Run the following commands to install all dependencies.
+Run the following commands in your project directory to install all dependencies:
+```bash
+> godep restore ./...
+```
+
+For further development - to install all project dependencies:
 ```bash
-> dep ensure
+> go get ./...
```

Comment:
`godep get` - what's the difference between "all dependencies" and "all project dependencies"?

Reply:
There's no `godep get`, `godep save` should get all missing dependencies, however it doesn't always work. `go get ./...` will usually do it better.